transport/ws/src/main/java/org/kaazing/gateway/transport/ws/AbstractWsBridgeSession.java

```diff
@@ -69,22 +70,22 @@
 
     protected ScheduledExecutorService scheduler;
     protected DefaultLoginResult loginResult;
-    protected final ActiveExtensions wsExtensions;
+    protected final ActiveWebSocketExtensions wsExtensions;
```

Comment:
Oh oops, must've been before I saw that presentation Todd forwarded a few weeks ago. Let's keep the shorter name.